index.html

```diff
@@ -91,10 +91,17 @@ <h1>Co-designing Inclusive Cities</h1><br />
 	<div id="ideasContainer">
 		<div class="ideas">
 			<h2>Ideas in Progress</h2>
-			<div class="ideasBlock">
+      <div class="ideas">
+        <h2>Ideas in Progress</h2>
+        <div class="ideasBlock">
```

Comment:
This file is (mostly) indented with tabs. Please intent your changes with tabs also, to make consistent.

I'm thinking we should switch the whole file to spaces at some point, but for now please use tabs to be consistent.